packages/components/src/components/notification/notification.stories.ts

```diff
@@ -134,37 +134,12 @@ export const ToastNotification = {
   name: 'Toast Notification (Default)',
   render: (_args: Record<string, any>) => {
     return html`
-      <div class="h-12 text-lg font-bold flex gap-1">
-        <button
-          class="top-right sd-interactive hover:text-white hover:opacity-75 bg-info w-24 text-white rounded-md"
-          notification-type="info"
-          variant="primary"
-        >
-          Info
-        </button>
-        <button
-          class="top-right sd-interactive hover:text-white hover:opacity-75 bg-success w-24 text-white rounded-md"
-          notification-type="success"
-          variant="primary"
-        >
-          Success
-        </button>
-        <button
-          class="top-right sd-interactive hover:text-white hover:opacity-75 bg-warning w-24 text-white rounded-md"
-          notification-type="warning"
-          variant="primary"
-        >
-          Warning
-        </button>
-        <button
-          class="top-right sd-interactive hover:text-white hover:opacity-75 bg-error w-24 text-white rounded-md"
-          notification-type="error"
-          variant="primary"
-        >
-          Error
-        </button>
+      <div class="flex gap-2">
+        <sd-button variant="secondary" notification-type="info" class="w-24 top-right"> Info </sd-button>
```

Comment:
Please use `data-notification-type` to make this element more valid.

And I'm not sure if the `top-right` class is needed but if it is instead please use `data-notification-position`.